stock/templates/registered/hrapplicationview.html

```diff
@@ -164,7 +164,7 @@ <h4 class="panel-title">
                                                 {% for comment in comments %}
                                                     <div class="panel panel-default">
                                                         <div class="panel-heading" role="tab" id="">
-                                                            <div class="panel-title">{{ comment.commenter_character.character_name }}
+                                                            <div class="panel-title">{{comment.commenter_character.created_on}} - {{ comment.commenter_character.character_name }}
```

Comment:
Update it on the branch you're working on - the pull request will update automatically.